bench/build.sbt

```diff
@@ -0,0 +1,7 @@
+enablePlugins(GTBenchmarkPlugin)
+
+libraryDependencies += "org.slf4j" % "slf4j-nop" % "1.7.25"
+
+jmhIterations := Some(5)
+jmhTimeUnit := Some("ms")
+jmhExtraOptions := Some("-jvmArgsAppend -Xmx8G")
```

Comment:
By hard coding these, we're assuming we'd never want to run the benches via say `jmh:run -t 1 -f 1 -wi 10 -i 10 .*Bench.*`? The `jmh:run` approach is definitely more "mystical".

Reply:
You can still do `jmh:run -t 1 -f 1 -wi 10 -i 10 .*Bench.*` if you want to; the plugin just allows some of these things to be set and managed individually so you could do:

```
sbt> set jmhIterations := Some(5)
sbt> bench
```

Reply:
PS: I think it's important to have a certain set of defaults version controlled to ensure you get somewhat consistent runs between analysis sessions (although the machine you run probably influences things heavily, so maybe it's false assumption of control).

Reply:
I think it's good to have sensible defaults.